release/conf/openmldb-env.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env bash
-export OPENMLDB_VERSION=0.8.2
+export OPENMLDB_VERSION=0.8.3
 # openmldb mode: standalone / cluster
 export OPENMLDB_MODE=${OPENMLDB_MODE:=standalone}
```

Comment:
Shall we set OPENMLDB_MODE default to cluster? maybe next pr.
It's difficult to use sbin/stop-tablets.sh or other scripts, we need to set OPENMLDB_MODE to env, or OPENMLDB_MODE=cluster sbin/xxx.sh

Reply:
yes, in most  case we deploy with cluster mode.

Reply:
+1

Reply:
#3511